src/VisualStudio/IntegrationTest/TestSetup/BindingAttributes.cs

```diff
@@ -4,8 +4,4 @@
 
 using Microsoft.VisualStudio.Shell;
 
-[assembly: ProvideCodeBase(CodeBase = @"$PackageFolder$\Microsoft.VisualStudio.IntegrationTest.Setup.dll")]
-[assembly: ProvideCodeBase(CodeBase = @"$PackageFolder$\Microsoft.VisualStudio.IntegrationTest.IntegrationService.dll")]
-[assembly: ProvideCodeBase(CodeBase = @"$PackageFolder$\Roslyn.Hosting.Diagnostics.dll")]
-
-[assembly: ProvideCodeBase(CodeBase = @"$PackageFolder$\Microsoft.Diagnostics.Runtime.dll")]
+// TODO: [assembly: ProvideCodeBase(CodeBase = @"$PackageFolder$\Microsoft.VisualStudio.IntegrationTest.Setup.dll")]
```

Comment:
>assembly [](start = 10, length = 8)

Why do we need code base for this assembly?

Reply:
I guess we'll see if integration tests fail


---
In reply to: [408478029](https://github.com/dotnet/roslyn/pull/43302#discussion_r408478029) [](ancestors = 408478029)

Reply:
Seems like this is required.